### ISSUE TITLE
barre meta 

### ISSUE THREAD

**Issue by leslieharicots** [marioncointre/larevanchedesharicots#12]
c;'est quoi la ba barre meta quil y a sur le cote droit ? 
est ce quil y a que nous qui le voyons ? 
si non je le ne veux pas stp

**Comment by marioncointre** [comment]
Non, c'est visible par tout le monde et c'est le nuage de mots clés si tu préfères
